backend/src/schema.ts

```diff
@@ -145,9 +154,9 @@ export const lists: Lists = {
         fields: {
             name: text({ validation: { isRequired: true } }),
             description: text({ validation: { isRequired: true } }),
-            medicine: relationship({ ref: "Medicine.study" }),
-            medicineAssignment: relationship({
```

Comment:
This relationship should be many (a study will have a therapeutic assignment for each participant, and each therapeutic assignment relates to a single study)